html-css/.hintrc

```diff
@@ -12,6 +12,7 @@
     "disown-opener",
     "html-checker",
     "meta-charset-utf-8",
-    "meta-viewport"
+    "meta-viewport",
+    "inline-style-disabled"
```

Comment:
I don't think this will work without pumping the version of the webhint. Have you tested it?

Reply:
@bolah2009 `no-inline-styles` is in [`@hint/configuration-web-recommended`](https://www.npmjs.com/package/@hint/configuration-web-recommended) ([see here](https://webhint.io/about/changelog/)), and I think webhint always uses its latest version of it, so when I tested it, it worked, I was however using the wrong name so I fixed that!